test/machine_learning/test_boosting_model.py

```
@@ -135,35 +130,27 @@ def test_full_data_check(self):
-                test["elastic_results"], test["config"], feature_ids, weighted_log_similarity_calculator=weight_log_sim
-            )
+            feature_ids = decision_maker.feature_ids
+            _boosting_featurizer = BoostingFeaturizer(test["elastic_results"], test["config"], feature_ids)
             if self.global_defect_type_model_folder.strip():
                 model = DefectTypeModel(create_filesystem(self.global_defect_type_model_folder))
                 model.load_model()
                 _boosting_featurizer.set_defect_type_model(model)
             gathered_data, _ = _boosting_featurizer.gather_features_info()
-            predict_label, predict_probability = test["decision_maker"].predict(gathered_data)
+            predict_label, predict_probability = decision_maker.predict(gathered_data)
             assert gathered_data == boost_model_results[str(idx)][0]
```

Ruff (0.12.2)

Use of `assert` detected

(S101)